settings.gradle.kts

```diff
@@ -12,13 +63,13 @@ develocity {
     }
 }
 val isCI: Boolean = System.getenv("CI") != null
+val isRelease: Boolean = System.getenv("CIRRUS_RELEASE") != null
 buildCache {
     local {
-        isEnabled = !isCI
+        isEnabled = !isCI && !isRelease
     }
     remote(develocity.buildCache) {
-        isEnabled = true
+        isEnabled = !isRelease
         isPush = isCI
```

Comment:
Did we effectively run into issue with the local or remote cache with the new release procedure?

Reply:
Let's remove those changes, and we will look later at develocity server to check the cache behavior.